src/healthkit.cs

```diff
@@ -2733,8 +2883,23 @@ interface HKQuantitySeriesSampleBuilder
 		[Wrap ("FinishSeries (metadata?.Dictionary, completionHandler)")]
 		void FinishSeries ([NullAllowed] HKMetadata metadata, HKQuantitySeriesSampleBuilderFinishSeriesDelegate completionHandler);
 
+		[Watch (6,0), iOS (13,0)]
+		[Async]
+		[Export ("finishSeriesWithMetadata:endDate:completion:")]
+		void FinishSeries ([NullAllowed] NSDictionary metadata, [NullAllowed] NSDate endDate, HKQuantitySeriesSampleBuilderFinishSeriesDelegate completionHandler);
+
+		[Watch (6,0), iOS (13,0)]
+		[Async]
+		[Wrap ("FinishSeries (metadata?.Dictionary, endDate, completionHandler)")]
+		void FinishSeries ([NullAllowed] HKMetadata metadata, [NullAllowed] NSDate endDate, HKQuantitySeriesSampleBuilderFinishSeriesDelegate completionHandler);
+
+
 		[Export ("discard")]
 		void Discard ();
+
+		[Watch (6,0), iOS (13,0)]
+		[Export ("insertQuantity:dateInterval:error:")]
+		bool InsertQuantity (HKQuantity quantity, NSDateInterval dateInterval, [NullAllowed] out NSError error);
```

Comment:
Would change it to insert, since we have a method that is already called insert and looks like an overload: 
```csharp
bool Insert (HKQuantity quantity, NSDate date, [NullAllowed] out NSError error);
```